sdk/samples/iot/paho_iot_hub_pnp_component_sample.c

```diff
@@ -85,7 +85,7 @@ static void subscribe_mqtt_client_to_iot_hub_topics(void);
 static void initialize_components(void);
 static void send_device_info(void);
 static void send_serial_number(void);
-static void request_device_twin_document(void);
+static void request_device_property_document(void);
```

Comment:
`property_document` isn't really a term i've seen used other places in PnP.  Do you mean something like `request_all_properties_from_service` (with a better name of course, i don't like the `from_service` specifically - maybe quset `request_all_properties` would be OK if i read this right.)